bleach/html5lib_shim.py

```diff
@@ -459,9 +459,22 @@ def convert_entity(value):
     if value[0] == "#":
         if len(value) < 2:
             return None
+
         if value[1] in ("x", "X"):
-            return six.unichr(int(value[2:], 16))
-        return six.unichr(int(value[1:], 10))
+            # hex-encoded code point
+            prefix, base = value[2:], 16
```

Comment:
I think I don't like the word "prefix" here. I was thinking it was the `#x` part, but it's not--it's the value as a string that's going to be converted later.

I don't want to reuse "value", but maybe `num_as_string`? Or `int_as_string`? Something like that?

Reply:
yeah good point renamed to `int_as_string`